public/app/features/explore/Logs/LogsFeedback.tsx

```diff
@@ -0,0 +1,38 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { Icon, Stack, useStyles2 } from '@grafana/ui';
+
+interface Props {
+  feedbackUrl?: string;
+}
+
+export function LogsFeedback({ feedbackUrl }: Props) {
+  const styles = useStyles2(getStyles);
+  return (
+    <Stack>
+      <a
+        href={feedbackUrl}
+        className={styles.link}
+        title="The metrics explorer is new, please let us know how we can improve it"
```

Comment:
fix

Reply:
Yeah we should! I created a chore for this here. https://github.com/grafana/grafana/issues/78813.